### ISSUE TITLE
IPFS.io is blocked

### ISSUE THREAD

**Issue by forever8896** [phantom/blocklist#1509]
Hello!

I understand the concern towards protecting users and their security, and whereas any file type can be hosted on IPFS, Its one of the most important infrastructure parts of the blockchain workflow, especially for NFTs. Since most NFTs are hosting their metadata off-chain, the best choice we currently have for distributed storage of data is the IPFS technology.

For developers, NFT peeps, and people within the crypto scene, I think blocking the ipfs site is limiting.

**Comment by Madeindreams** [comment]
Yeah what is going with this? Just found out as well. Please remove this ASAP.

**Comment by Madeindreams** [comment]
Where is it even listed at. I can't find it?!

**Comment by IAmSalvaMartini** [comment]
@Harii94 https://ipfs.io/ipfs/QmPmZhB2hAdP8pJmwkherssDfneNYHtRiTJyTMfyHbnRm3

**Comment by Madeindreams** [comment]
I have solved the problem! I have uninstalled this wallet!

**Comment by Madeindreams** [comment]
> I think its because a lot of crypto scams are hosted on ipfs and if you contact the [abuse@ipfs.io](mailto:abuse@ipfs.io) then they do nothing about it.

There is nothing they can do about it just like there is nothing they can do about the content on chain. IPFS is decentralized. By doing this, Phantom block access to decentralization. Also worth noting. If a project deployed on IPFS is a scam, the content is a hash so it's pretty easy to block specific content. Not en entire ecosystem. And why not block the actual address. 

Anyways i'm done with this product.
